cobra/internal/base/moimpl.py

```diff
@@ -447,6 +483,31 @@ def __init__(self, parentMoOrDn, markDirty, *namingVals,
             # pylint:disable=protected-access
             self.__parentMo.__modifyChild(self, attach=True)
 
+    def clone(self, parentMo=None):
```

Comment:
Wouldn't we be able to define that behavior in our own implementation? My thought here is just to avoid creating a new method for something that is already part of stdlib.